### ISSUE TITLE
Need flotsam & scrap metal pickup objects

### ISSUE THREAD

**Issue by RossMcConeghy** [TwoClunkers/Clunk-Genesis#17]
cogs, bolts, metal scraps, etc...
modeled so far:
![image](https://cloud.githubusercontent.com/assets/4841483/7039169/7326cff6-dd72-11e4-94ec-e9041842c846.png)

**Comment by jesterstud** [comment]
Awesome! what do you think about having gears and bolts as inventory items?

**Comment by RossMcConeghy** [comment]
Yeah, I'd think of them as low level crafting components, like "Bolts & Nuts", "Cogs" like multiple in a pile kinda thing.  
Textured flotsam:  
![image](https://cloud.githubusercontent.com/assets/4841483/7108508/a167649c-e13d-11e4-81fa-1748a3ab8f50.png)
  
Made every bit of this, took every picture for every texture, did composites in Gimp. Now I gotta bring 'em into Unity and see how they look. =D

**Comment by jesterstud** [comment]
Woot! wish I could say I fixed the serialization... but I'm very very close
now that the chicken coop is decent

**Comment by RossMcConeghy** [comment]
Okk, the flotsam has been brought into Unity and set up with convex mesh colliders and sphere trigger colliders, they just lack scripting to act as "Pickups".

**Comment by jesterstud** [comment]
fixed serialization - i looked at the chunk class and decided i no longer
like the idea of a "second list". the reason is that we are constantly
going to be referencing "cubespace" anyway to see if objects can be
spawned. so having a whole new set of management tools to save, load, list
... daunting. lemme try a linked list in the air voxel.
